pkg/controller/jobframework/integrationmanager.go

```diff
@@ -61,6 +61,9 @@ type IntegrationCallbacks struct {
 	GVK schema.GroupVersionKind
 	// NewReconciler creates a new reconciler
 	NewReconciler ReconcilerFactory
+	// NewAdditionalReconcilers creates a new additional reconcilers
+	// (this callback is optional)
```

Comment:
```suggestion
	// NewAdditionalReconcilers creates additional reconcilers (optional)
```
nit

Reply:
In other places we start from a new line https://github.com/kubernetes-sigs/kueue/blob/787758bfa570fd7972ded660f60b2a743cc12963/pkg/controller/jobframework/integrationmanager.go#L59-L60. I would prefer to have the same to consistency. WDYT?

Reply:
Ah, sure, I didn't notice that, let's keep them consistent. Still, I would prefer to say just "NewAdditionalReconcilers creates additional reconcilers" (without "new", as it seems obvious as we create new objects).

Reply:
Done